wfinterop/wes/client.py

```diff
@@ -74,8 +74,7 @@ def ListRuns(self):
     def RunWorkflow(self, request, parts=None):
         return self._wes_client.run(wf=request['workflow_url'],
                                     jsonyaml=request['workflow_params'],
-                                    attachments=request['attachment'],
-                                    parts=parts)
+                                    attachments=request['attachment'])
```

Comment:
@pr80ik - I'm curious what you mean by "unexpected." This argument is needed to build the WES request within the orchestrator and pass it to `wes_client` (enabled by the changes in [this fork/branch of 'workflow-service'](https://github.com/jaeddy/workflow-service/tree/delegate-wes-parts). Without it, the request will be constructed using `wes_client.util::build_wes_request` (which I can't control as easily).

However, if you're getting an error, that's something I should look into.